manager/src/main/java/com/redhat/service/bridge/manager/connectors/ConnectorsServiceImpl.java

```diff
@@ -63,9 +70,26 @@
 
         Connector connector = callConnectorService(connectorType, connectorPayload, newConnectorName);
 
+        newConnectorEntity.setConnectorExternalId(connector.getId());
+
         return Optional.of(newConnectorEntity);
     }
 
+    @Override
+    public void deleteConnectorIfNeeded(Processor processor) {
+
+        List<ConnectorEntity> connectors = connectorsDAO.findByProcessorId(processor.getId());
```

Comment:
@robpblake currently not, in fact it's a 1-1 relation mapped as a 1-N. We can change the DB if we prefer

Reply:
@lucamolteni Yeah, let's make the API of the `ConnectorsDAO` return what we expect, so in this case a single `ConnectorEntity` rather than the `List<ConnectorEntity>` 👍